src/backend/marsha/core/tests/test_api_timed_text_track.py

```diff
@@ -189,15 +189,15 @@ def test_api_timed_text_track_read_list_anonymous(self):
         self.assertEqual(response.status_code, 401)
 
     def test_api_timed_text_track_read_list_token_user(self):
-        """A token user associated to a video is not able to read a list of timed text tracks."""
+        """A token user associated to a video is able to read a list of timed text tracks."""
         timed_text_track = TimedTextTrackFactory()
         jwt_token = AccessToken()
         jwt_token.payload["video_id"] = str(timed_text_track.video.id)
 
         response = self.client.get(
             "/api/timedtexttracks/", HTTP_AUTHORIZATION="Bearer {!s}".format(jwt_token)
         )
-        self.assertEqual(response.status_code, 405)
+        self.assertEqual(response.status_code, 200)
```

Comment:
I think you should test the content of the response.

Reply:
Is it enough like this?

Reply:
I would have added 2 timedtexttracks and checked the presence of both their ids in the response... sometimes things can get filtered or permissioned or duplicated or... ?

Reply:
done